basics.js

```diff
@@ -1,24 +1,85 @@
 /* Create a `myName` variable and assign it a String value */
 
+function Person (name) {
+
+    var myName = name;
+   return myName;
+ }
+  console.log(Person('Gail'));
+
+
 /* Create a `person` variable and give it 2 properties,
  * `name`, assign it the same name as before,
  * as well as an `age` (number);
  */
+var Person = {
+   name: 'Gail',
+   age: 21,
+};
+
 
 /* Create a function called `greet`,
  * it should take a 1 parameter, `name`
  * and it should print "Hello, my name is {name}"
  */
+function greet (name) {
+return "Hello, my name is " + name;
 
+}
+console.log(greet('Joe'));
```

Comment:
`console.log` is okay for testing locally. But please delete them before committing.